docs/CHANGELOG.asciidoc

```diff
@@ -28,6 +28,359 @@ This section summarizes the changes in each release.
+* Bumps numeral.js to fix negative time formatting bug {pull}35298[#35298]
+
+Monitoring::
+* Improves performance of the Logstash Pipeline Viewer {pull}31293[#31293]
+* Updates tests for Monitoring Logstash Pipeline Viewer performance improvements {pull}33570[#33570]
+* Sends encrypted telemetry data to telemetry service {pull}33121[#33121]
+* Checks for 6.7.2 instead of 6.7.3 in Telemetry {pull}35814[#35814]
+* Introduces Logs UI {pull}31275[#31275]
+* Determines the Metricbeat migration status logic {pull}34871[#34871]
+* Forces collectors to indicate when they are ready in Montoring/Telemetry {pull}36153[#36153]
+* Makes change so that once the buffer has events, the collector is always ready {pull}36995[#36995]
+* Fixes 500 error from /api/stats {pull}36986[#36986]
+* Removes improper usage of query {pull}37099[#37099]
+* Uses fixed_interval explicitly for date_histogram aggregations {pull}37023[#37023]
+* Creates license check only when there’s a separate monitoring cluster {pull}33590[#33590]
+* Adds uiCapabilities.logs check {pull}35562[#35562]
+* Creates a simple cancelable promise, and adds  basic support for Typescript {pull}36293[#36293]
+* Fixes default message if value is invalid {pull}36737[#36737]
+* Fixes sorting of Monitoring nodes on CPU, Throttle, and Heap columns {pull}36125[#36125]
+* Swaps the anchor tag with EuiLink {pull}37020[#37020]
+* Typecasts the attr boolean to the correct type {pull}37031[#37031]
+* Fixes CPU calculation for APM {pull}37230[#37230]
+* Optimizes content to be more vertical for resizing consistency {pull}37120[#37120]
+* Uses i18n instead of injectI18n {pull}37845[#37845]
+* Adds a link to License Management {pull}37771[#37771]
+
+Operations::
+* Sets branch in package.json to 7.2 {pull}37446[#37446]
+* Reverts [kbn/es] pin 7.x snapshot to elastic/elasticsearch#39582 i… {pull}32580[#32580]
+* Exits if starting Kibana as root {pull}21563[#21563]
+* Derives bootstrap cache path from base branch, makes optional {pull}32793[#32793]
+* Uses default keys when running from source {pull}36452[#36452]
+* Allows running --dev without kibana.dev.yml {pull}36400[#36400]
+* Reenables github checks reporter {pull}37280[#37280]
+* Fixes inefficiencies in the dev/license_checker module and rewrites it in Typescript {pull}37807[#37807]
+* Enables the use of the cache-loader across all the environments {pull}35406[#35406]
+* Adds exception to transpile normalize-url mode to es5 when bundled into the DLL {pull}35804[#35804]
+* Uses script to clean Storybook DLL {pull}35243[#35243]
+* Upgrades jQuery to 3.4.1 {pull}35908[#35908]
+* Enforces all packages to use the same Typescript version {pull}32440[#32440]
+* Fixes plugin generator package.json paths {pull}33728[#33728]
+
+Platform::
+* Fixes version for plugin showing as undefined {pull}36219[#36219]
+* Updates Japanese translation files {pull}37993[#37993]
+* Adds Japanese locale {pull}36940[#36940]
+* Adds Japanese translations {pull}37229[#37229]
```

Comment:
Only keep the Adds Japanese Local PR and remove the rest of Localization related PRs